paddle/fluid/operators/detail/grpc_server_test.cc

```diff
@@ -138,3 +137,8 @@ TEST(PREFETCH, CPU) {
     EXPECT_EQ(ptr[0 + i * value.dims()[1]], static_cast<float>(i * 2));
   }
 }
+
+TEST(PREFETCH, CPU) {
+  std::shared_ptr<detail::RPCClient> client(new detail::GRPCClient);
```

Comment:
why shared_ptr? would unique_ptr be enough?

Reply:
Done.